library.py

```diff
@@ -34,7 +34,6 @@
 
 PLOT_ENABLE = True
 
-
```

Comment:
That was an oversight of my part while debugging, sorry. It shouldn't be absent indeed